build-test.xml

```diff
@@ -62,6 +63,7 @@
             classpathref="classpath"
             verbose="false"
             debug="true"
+            debuglevel="lines,vars,source"
```

Comment:
should the UCSC guys need to add columns here, in future, whenever they are ready :)